docs/content/providers/kubernetes-ingress.md

```diff
@@ -64,23 +64,36 @@ as is a common pattern in the kubernetes ecosystem.
 When using a single instance of Traefik with LetsEncrypt, no issues should be encountered,
 however this could be a single point of failure.
 Unfortunately, it is not possible to run multiple instances of Traefik 2.0 with LetsEncrypt enabled,
```

Comment:
```suggestion
When using a single instance of Traefik Proxy with Let's Encrypt, you should encounter no issues.
However, this could be a single point of failure.
Unfortunately, it is not possible to run multiple instances of Traefik 2.0 with Let's Encrypt enabled,
```